### PR TITLE
Fixes issue #69 (GSL interface) and #338 (MixtureConstraintEntry)

### PR DESCRIPTION
**Pull request by dvandyk**

### CODE REVIEW

File: eos/constraint_TEST.cc [hunk 1]
```diff
@@ -1134,6 +1134,139 @@ class ConstraintDeserializationTest :
...
+                TEST_CHECK_EQUAL(1, blocks.size());
+
+                Parameters p = Parameters::Defaults();
+                LogLikelihood llh(p);
+                llh.add(c);
+
+                // evaluation off mode
+                p["mass::b(MSbar)"] = 4.6;
+                p["mass::c"] = 1.3;
+                TEST_CHECK_NEARLY_EQUAL(llh(), -4.779399451, 1e-8);
```
Comment (mreboud) [inline eos/constraint_TEST.cc:1267]: This is a value you get from the code I guess? Or could you check it against two multivariate Gaussian distributions?

File: eos/utils/gsl-interface_TEST.cc [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        virtual void run() const
+        {
+            // default construction
+            {
+                GSLMatrixPtr();
+            }
+
+            // allocation with alloc
+            {
+                GSLMatrixPtr p(gsl_matrix_alloc(10, 10));
```
Comment (mreboud) [inline eos/utils/gsl-interface_TEST.cc:44]: If I understand correctly this is also indirectly testing the destructors, right?

**Review by mreboud — APPROVED** [review]

File: eos/constraint_TEST.cc (hunk 1 above)
Comment (dvandyk) [reply eos/constraint_TEST.cc:1267]: Did that!

Comment (mreboud) [reply eos/constraint_TEST.cc:1267]: Excellent :)

File: eos/utils/gsl-interface_TEST.cc (hunk 2 above)
Comment (dvandyk) [reply eos/utils/gsl-interface_TEST.cc:44]: Yes. The ``MatrixDeleter`` is used.

**Pull request merged**